File: easy/cheering/cheering.kt
fun cheering(yards:Int): String{

	if (yards > 10){
		return("High Five")
    }
	else if(yards < 1){
		return("shh")
    }
	var ra = ""
    for (i in 1..yards) {

        ra = ra + "Ra!"
    }
    return(ra)
}

fun main(args: Array<String>){
	
    println("Number of yards : ")
    var yards = readLine()!!.toInt()

   	println(cheering(yards))

}

File: easy/cheering/cheering.php
<!DOCTYPE html>
<html>
<head>
	<title>Cheering</title>
</head>
<body>
	<form  method="post">
		<label for="yards">Number of yards: </label>
		<input type="text"  name="yards">

		<input type="submit" value="Action" >
	</form>	
	<br>
	<?php

		function cheering($yards){
			if ($yards == NULL ){
				return("");
			}

			if ($yards > 10){
				echo "High Five";
		    }
			else if($yards < 1){
				echo "shh";
		    }
		    else{
		    	$ra = "";

				for ($i=0; $i < $yards; $i++) { 
					echo "Ra!";
				}

		    }

		}

	    $yards = $_POST["yards"];

	    cheering($yards);
        
	?>

</body>
</html>
		
